.github/workflows/ci-conda.yml

```diff
@@ -90,12 +90,14 @@ jobs:
         run: |
           ./bootstrap
           echo "::add-matcher::.github/workflows/configure-systempackage-problem-matcher.json"
-          ./configure --enable-build-as-root --with-python=$CONDA_PREFIX/bin/python --prefix=$CONDA_PREFIX $(for pkg in $(./sage -package list :standard: --has-file spkg-configure.m4 --has-file distros/conda.txt); do echo --with-system-$pkg=force; done)
+          ./configure --enable-build-as-root --with-python=$CONDA_PREFIX/bin/python --prefix=$CONDA_PREFIX $(for pkg in $(./sage -package list :standard: --has-file spkg-configure.m4 --has-file distros/conda.txt); do if [ $pkg != "rpy2" ]; then echo --with-system-$pkg=force; fi; done)
           echo "::remove-matcher owner=configure-system-package-warning::"
           echo "::remove-matcher owner=configure-system-package-error::"
 
           # Manually install ptyprocess for now, until https://github.com/sagemath/sage/issues/32147 is fixed
           pip install --no-build-isolation -v -v ptyprocess==0.5.1
+          # and lrcalc until https://github.com/conda-forge/lrcalc-feedstock/issues/12 is fixed
+          pip install --no-build-isolation -v -v lrcalc==2.1
 
```

Comment:
Why are these manual "pip install"s necessary? Is this not taken care of in the generated `environment.yml`?

Reply:
There is no ptyprocess 0.5.1 on conda-forge (oldest version is 0.5.2), and lrcalc actually distributes 2.0 under the 2.1 tag.

Reply:
Yes, but we do have the pip installs in the environment.yml.

Reply:
What do you mean? This is here only to make the check pass that all dependencies are correctly specified in the conda files. For a normal installation, `pip install --no-build-isolation --no-deps -v -v -e ./src`  takes care to install all python dependencies.

Reply:
If you just remove `build/pkgs/lrcalc_python/distros/conda.txt` and `build/pkgs/ptyprocess/distros/conda.txt` (which is comments only at the moment), then `bootstrap-conda` will create `pip` lines in `src/environment.yml` -- just like the ones you see at the end of `src/environment-optional.yml`.

Reply:
But I don't think it's necessary to change the environment files.

Reply:
They are generated. You change the source, so that such ad-hoc command do not have to be added.

Reply:
As I said above, these adhoc commands are only necessary in the context of the conda-workflow and I thus don't see an issue with a "localized fix".

Reply:
No, the workflow only tests what we recommend to the user in our installation manual.
So by changing it here ad-hoc, you will miss that the instructions are broken.

Reply:
No, the `--no-deps` argument below is not recommended for users (neither is the `pip check` command run later). This is only here to make sure that our setup is correct.

Reply:
Here in the CI, `lrcalc` will be installed once using the conda env; then `configure` tests for it successfully; then you override it with a pip install.

In the user instructions, the last step is missing.

Reply:
Installing the wrong versions of these packages first using conda is clearly unnecessary; we would not want to recommend that to users, and so that's also not what we should test.

Reply:
Feels like unnecessary optimization to me.

Reply:
Yes, I know you are a big fan of this kind of optimization...I'm not.